src/hooks/use-snapshots.tsx

```diff
@@ -193,12 +193,6 @@ export const SnapshotProvider: React.FC< { children: ReactNode } > = ( { childre
 		);
 	}, [] );
 
-	useEffect( () => {
-		if ( initiated && ! loadingServerSnapshots && allSnapshots ) {
-			clearFloatingSnapshots( allSnapshots );
```

Comment:
Let's also remove the definition of `clearFloatingSnapshots` function which is not used anymore.

Reply:
Yeah, that's a good point. I was considering it, but left it there in case it is needed elsewhere. But it is a good call to remove it. 🙂 Will do it tomorrow.

Reply:
I have removed the definition in https://github.com/Automattic/studio/pull/937/commits/773722c214c14645eac2cd6d037a00ea721259e2.